internal/core/service/room_service_test.go

```
@@ -0,0 +1,308 @@
+	createRoomCalled   bool
+	createRoomJoinRule string
+	createRoomType     string
+	createRoomErr      error
+
+	// UpdateRoomState captures
+	updateRoomCalled   bool
+	updateRoomJoinRule string
+	updateRoomErr      error
+}
+
+func (m *mockMatrixPort) Connect(_ context.Context) error    { return nil }
```

golangci-lint (2.11.3)

[error] File is not properly formatted

(gofmt)